compiler/docgen.nim

```diff
@@ -1042,12 +1042,12 @@ proc genOutFile(d: PDoc): Rope =
       rope(getClockStr()), code, d.modDeprecationMsg])
   if optCompileOnly notin d.conf.globalOptions:
     # XXX what is this hack doing here? 'optCompileOnly' means raw output!?
-    code = ropeFormatNamedVars(d.conf, getConfigVar(d.conf, "doc.file"), ["title",
-        "tableofcontents", "moduledesc", "date", "time",
+    code = ropeFormatNamedVars(d.conf, getConfigVar(d.conf, "doc.file"), ["nimdoccss",
+        "title", "tableofcontents", "moduledesc", "date", "time",
         "content", "author", "version", "analytics", "deprecationMsg"],
-        [title.rope, toc, d.modDesc, rope(getDateStr()),
-                     rope(getClockStr()), content, d.meta[metaAuthor].rope,
-                     d.meta[metaVersion].rope, d.analytics.rope, d.modDeprecationMsg])
+        [rope($relativeTo(d.conf.outDir / RelativeFile"nimdoc.out.css", d.destFile.splitFile().dir)),
```

Comment:
nit: for readability, maybe introduce a variable eg:
```
let cssHref = $relativeTo(d.conf.outDir / RelativeFile"nimdoc.out.css", d.destFile.splitFile().dir
```

Reply:
fair enough, but its only used once anyways :)

Reply:
now i see u also have `[rope($relativeTo(conf.outDir / RelativeFile"nimdoc.out.css", filename.splitFile().dir)),`
maybe then refactor as a proc:
`getCssHref(outDir: string, path: string): string`
called twice